publish.js

```diff
@@ -124,36 +124,57 @@ publish.convertToISO8601 = function (timestamp) {
 };
 
 /**
+ * Calls publish.execSync with a command crafted from a template string.
+ * If it is a test run, the command will only be logged to the console.
+ *
+ * @param cmdTemplate {String} - A string template of the command to execute.
+ *                               Can provide tokens in the form ${tokenName}
+ * @param values {Object} - the tokens and their replacement.
+ *                          e.g. {tokenName: "value to insert"}
+ * @param hint {String} - A string template of a hint for recovering from an error thrown by the executed command.
+ *                               Can provide tokens in the form ${tokenName}
+ * @param isTest {Boolean} - indicates if this is a test run or not. If it is
+ *                           a test run, the command will be logged but not executed.
+ */
+publish.execSyncFromTemplate = function (cmdTemplate, cmdValues, hint, isTest) {
+    var cmdStr = es6Template(cmdTemplate, cmdValues);
+    publish.log("Executing Command: " + cmdStr + "\n");
```

Comment:
I think the "\n" character is unnecessary here since console.log already starts a new line